packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -450,11 +470,15 @@ const AccountTransferForm = observer(
                                         >
                                             <Dropdown
                                                 id='transfer_from'
-                                                className='account-transfer-form__drop-down'
+                                                className={classNames('account-transfer-form__drop-down', {
+                                                    'account-transfer-form__drop-down--disabled':
+                                                        is_poa_or_poi_pending_for_mf,
+                                                })}
```

Comment:
same concern, not sure why we need another class to define a style for the disabled dropdown, `disabled` should handle it properly.

Reply:
replied in the aboveeee